src/main/java/net/io/kino/controller/LogsController.java

```diff
@@ -0,0 +1,44 @@
+package net.io.kino.controller;
+
+import net.io.kino.model.loggingaction.EventData;
+import net.io.kino.model.loggingaction.EventType;
+import net.io.kino.service.logger.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@RestController
+@RequestMapping("/logs")
+public class LogsController {
+
+    Logger logger;
+
+    private LogsController(@Autowired Logger logger) {
+        this.logger = logger;
+    }
+
+    @GetMapping("/")
+    public List<EventData> getAllLogs() {
+        return logger.getAllLogs();
+    }
+
+    @GetMapping("/after/")
```

Comment:
Nie musiałaś robić kontrollerów do tego, bo robi to Jacek, ale jak już zrobiłaś to:

Jak projektuje się API restowe to w ścieżce umieszcza się resources jakich operacja dotyczy, a takie parametry filtrowania w query params.

Przykładowo:
/users - zwraca użytkowników
/users/1 - zwraca użytkownika o id = 1
/users/1/products - zwraca produkty użytkownika o id = 1

Ale gdy chcemy jakieś filtrowanie dodać do wyników to robimy to jako:
/users?after=xxx
bo to jest tamta lista userów, tylko że z dodatkowymi ograniczeniami. Dzięki temu widać dokładnie hierarchie i co jest zasobem a co nie.

Reply:
Usunąć całą klasę w takim razie skoro Jacek to robi?

Reply:
Może usuń. Jacek zrobi po swojemu i poprawi